app/frontend/helpers/hashCode.js

```diff
@@ -0,0 +1,19 @@
+const hashCode = (s) => {
```

Comment:
please cite source if you found it somewhere. Is it some standard hash algorithm? If so, please call the function to reflect that.

Reply:
https://gist.github.com/iperelivskiy/4110988